spotify_tensorflow/luigi/tfdv.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import logging
+import tempfile
+
+import luigi
+from spotify_tensorflow.luigi.tfx_task import TFXBaseTask
+from spotify_tensorflow.luigi.utils import fetch_tfdv_whl, get_uri
+import spotify_tensorflow.pipeline.tfdv as tfdv_pipeline
+import tensorflow_data_validation as tfdv
+
+logger = logging.getLogger("luigi-interface")
+
+
+class TFDVGenerateStatsTask(TFXBaseTask):
+    """
+    Thin wrapper around TensorFlow Data Validation's `tfdv.generate_statistics_from_tfrecord`.
+
+    By default, the resulting stats is written as a `_stats.pb` file inside each respective
+    input target path. This ensures the stats object is co-located with the data.
+    """
+
+    python_script = tfdv_pipeline.__file__
+
+    def __init__(self):
+        super(TFDVGenerateStatsTask, self).__init__()
+        if not self.local_runner:
+            self.requirements_file = TFDVGenerateStatsTask._construct_reqs_txt()
+
+    def tfx_args(self):
+        args = []
+        if not self.local_runner:
+            # Fetch the TFDV wheel ourselves for the Dataflow worker.
+            local_tfdv_whl_path = fetch_tfdv_whl(version=tfdv.__version__)
```

Comment:
Maybe add a comment here about why we need to fetch tfdv wheel. I guess google is going to fix it, right?